cli/azd/cmd/extension.go

```diff
@@ -217,10 +218,29 @@ func (a *extensionListAction) Run(ctx context.Context) (*actions.ActionResult, e
 			Name:      extension.DisplayName,
 			Namespace: extension.Namespace,
 			Version:   version,
+			Source:    extension.Source,
 			Installed: installedExtensions[extension.Id] != nil,
 		})
 	}
 
+	if len(extensionRows) == 0 {
+		if a.flags.installed {
+			fmt.Println(output.WithWarningFormat("WARNING: No extensions installed.\n"))
```

Comment:
Curious about why you are using `fmt.Println`, I think we don't use. IIRC, we decided to use `fmt.Fprintln()` so we could set the io.Writer to control the `--output json` or things like that.

Can you use the `Console.Message()` here?

Reply:
Will update to use console.message.  Thanks for the reminder 🙏